Tribler/Core/CacheDB/SqliteCacheDBHandler.py

```diff
@@ -2456,12 +2478,15 @@ def getMostPopularChannels(self, max_nr=20):
               "FROM Channels ORDER BY nr_favorite DESC, modified DESC LIMIT ?"
         return self._getChannels(sql, (max_nr,), includeSpam=False)
 
-    def getMySubscribedChannels(self, include_dispersy=False):
-        sql = "SELECT id, name, description, dispersy_cid, modified, nr_torrents, nr_favorite, nr_spam " + \
+    def getMySubscribedChannels(self, include_dispersy=False, first=0, last=None, count = False):
+        sql = "SELECT (id) " if count else "SELECT id, name, description, dispersy_cid, modified, nr_torrents, nr_favorite, nr_spam " + \
```

Comment:
`COUNT` keyword is missing, is it?
`SELECT COUNT (id) " if count else` ...

Reply:
Yeah, you right. I wonder how it was able to work before...

Reply:
Oh! I stopped using it in some version, so it was not really necessary.